pkg/ccl/backupccl/targets.go

```diff
@@ -257,6 +257,11 @@ func fullClusterTargets(
 				fullClusterDBs = append(fullClusterDBs, dbDesc)
 			}
 		case catalog.TableDescriptor:
```

Comment:
before this diff and Marius's, we were including offline tables in cluster backups. Was that intended? @dt

Reply:
I don't think so (cc @adityamaru in case I'm forgetting something).

Reply:
Not that I can recall, unless the presence of the descriptors was something that affected our ability to roll back the import on restore?

Reply:
Ah, yeah, that was probably it -- we probably included the offline tables because restored jobs referred to them.

Reply:
So, since we're no longer backing up jobs in cluster restores, are we cool with only backing up offline tables, if they're apart of an import rollback? But maybe that will screw with other offline descriptors that have in-progress schema change jobs?

Maybe I'll keep offline tables in the cluster for now, and audit this logic during stability.

Reply:
will remove offline tables after andrew removes offline state from schema change job